Makefile

```diff
@@ -72,3 +73,19 @@ check-french-spacing: $(DOC_TEX_ALL_SORTED)
 checkbiw: $(DOC_TEX_ALL_SORTED)
 	$(QUIET)$(CHECKBIW) -v -c $+
 
+stats: pdf
+	$(QUIET)echo "\e[1mThesis Stats:\e[0m"
+	$(QUIET)echo "  \e[1mdetexed sources and removed empty lines:\e[0m"
+	$(QUIET)echo "    lines:      $(shell detex $(DOC_TEX_ALL_SORTED) | sed '/^$$/d' | wc -l)"
+	$(QUIET)echo "    words:      $(shell detex $(DOC_TEX_ALL_SORTED) | sed '/^$$/d' | wc -w)"
+	$(QUIET)echo "    characters: $(shell detex $(DOC_TEX_ALL_SORTED) | sed '/^$$/d' | wc --chars)"
+	$(QUIET)echo
+	$(QUIET)echo "    Please note: There are possibly deviations compared to the final PDF as the compiled"
+	$(QUIET)echo "                 version might contain additional words, such as 'in Figure 3.5 on the"
+	$(QUIET)echo "                 next page' or the bibliography."
+	$(QUIET)echo
+	$(QUIET)echo "  \e[1mCompiled PDF:\e[0m"
+	$(QUIET)echo "    pages: $(shell pdfinfo $(DOC_PDF) | awk '/^Pages:/ {print $$2}') (total, not content pages)"
+	$(QUIET)echo "    words: $(shell pdftotext $(DOC_PDF) $(DOC_TEX:.tex=.txt) | cat $(DOC_TEX:.tex=.txt) | wc -w)"
```

Comment:
I think we should add the *.txt file that got created here to the clean target, so that it gets removed again.

Reply:
fixed